### PR TITLE
build: add eslint-prettier-airbnb

### PR DESCRIPTION
**Pull request by ottonotto**
refactor: app.js
build: add .eslintrc, .prettierrc, package.json, package-lock.json

### CODE REVIEW

**Comment by shams-ali** [comment]
plesae fix the title...i dont know how you got upstream there?

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        "react-hooks/exhaustive-deps":          "warn",
+        "react-hooks/rules-of-hooks":           "error",
+        "react/jsx-filename-extension":         [1, { "extensions": [".js", ".jsx"] }],
+        "react/jsx-indent-props":               [2, 4],
+        "react/jsx-indent":                     [2, 4],
+        "react/jsx-one-expression-per-line":    [0],
+        "react/prefer-stateless-function":      [1],
+        "react/static-property-placement":      [1, "property assignment"]
+    }
+}
```
Comment (shams-ali) [inline .eslintrc:33]: end of line....

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "printWidth": 100,
+    "semi": true,
+    "singleQuote": true,
+    "tabWidth": 4,
+    "trailingComma": "es5"
+}
```
Comment (shams-ali) [inline .prettierrc:7]: end of line

File: src/App.js [hunk 3]
```diff
@@ -1,25 +1,26 @@
...
+                    href="https://reactjs.org"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                >
+                    Learn React
+                </a>
+            </header>
+        </div>
+    );
+};
```
Comment (shams-ali) [inline src/App.js:24]: implicit return...this rule should be in your eslint...prettier shuoldhave auto done this for you

File: package.json [hunk 4]
```diff
@@ -34,5 +34,17 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "eslint": "^8.5.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-airbnb-base": "^15.0.0",
```
Comment (shams-ali) [inline package.json:41]: you have both airbnb and airbnb-base....but your only using airbnb in your .eslint...i dont see airbnb-base

Comment (ottonotto) [reply package.json:41]: yea, i need to do what you tell me to do..and read what each thing does.. airbnb-base is for JS w/o react plug-ins.. which i will be using

File: src/App.js (hunk 3 above)
Comment (ottonotto) [reply src/App.js:24]: still, trying to find rules for implicit return. what i have tried isnt workin. but will make it happen capt

File: package.json (hunk 4 above)
Comment (ottonotto) [reply package.json:41]: i added airbnb-base to .eslint.

File: .prettierrc (hunk 2 above)
Comment (ottonotto) [reply .prettierrc]: fixed

File: src/App.js (hunk 3 above)
Comment (ottonotto) [reply src/App.js]: fixed

File: package.json (hunk 4 above)
Comment (shams-ali) [inline package.json:41]: not needed

File: .eslintrc [hunk 5]
```diff
@@ -0,0 +1,36 @@
+{
+    "parser": "@typescript-eslint/parser",
+    "extends": [
+        "airbnb",
+        "airbnb-base",
```
Comment (shams-ali) [inline .eslintrc:5]: not needed

**Review by shams-ali — APPROVED** [review]

**Pull request merged**